trace_http_client_test.go

```diff
@@ -20,6 +20,7 @@ func TestTraceHTTPClient(t *testing.T) {
 		desc         string
 		errExpected  bool
 		service      twirptest.Haberdasher
+		clientOpts   []TraceClientOpt
```

Comment:
Did you mean for this to be `ClientOpt`?
```suggestion
		clientOpts   []ClientOpt
```